extlibs/headers/mingw/dinput.h

```diff
@@ -0,0 +1,2467 @@
+/*
+ * Copyright (C) the Wine project
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
```

Comment:
I assume LGPL headers are fine to be shipped, right?

Reply:
We are only working around a toolchain "bug". Either you use the one you already have or you use the one we provide. In both cases you are `#include`ing an LGPL header.